### ISSUE TITLE
console errors

### ISSUE THREAD

**Issue by slanted** [eric-manwill/nuskin-sales-platform-demo#1]
Hey nice job @eric-manwill. So when I do a npm install and then webpack the project and pull it up in a browser I still get some errors. I know you spoke to me about still having dependencies on the leftnav and the header. Is this still something you're working on?

**Comment by eric-manwill** [comment]
@slanted Yes, that is what I'm working on now. I'm also running into a problem with NSJS assuming it can request configuration data from the server the page is being served from. I've not tackled that problem yet, as you mentioned recently that you were doing some more retooling of the code bundle.
